platform/features/table/res/templates/scrolling.html

```diff
@@ -0,0 +1,9 @@
+<div ng-controller="RTTelemetryTableController">
+    <mct-table
+        headers="headers"
+        rows="rows"
+        enableFilter="true"
+        enableSort="true"
+        auto-scroll="true">
```

Comment:
Hrm, I don't see this template used in any bundles here-- can you remove it or let me know that I'm missing some other thing that uses this?